olive/evaluator/evaluation.py

```diff
@@ -307,7 +318,7 @@ def get_user_config(config: dict):
     eval_func = getattr(config, "evaluate_func", None)
     eval_func = user_module.load_object(eval_func)
 
-    if not dataloader and not eval_func:
+    if config.input_names and config.input_shapes and config.input_types and not dataloader and not eval_func:
```

Comment:
current DummyDataloader will generate data in type float32 if config.input_types is None.

Reply:
Removed input_types checking.